<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasurementDialog</class>
 <widget class="QDialog" name="MeasurementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pomiar</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Radionuklid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nuclide_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Obiekt pomiaru</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="measurementObject_lineEdit">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stop_pushButton">
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Źródło</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="sources_progressBar">
       <property name="value">
        <number>24</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="format">
        <string>%v z %m</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Pkt. pomiaru</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="points_progressBar">
       <property name="value">
        <number>24</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="format">
        <string>%v z %m</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Powtórzenie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="repeating_progressBar">
       <property name="value">
        <number>24</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="format">
        <string>%v z %m</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Czas pomiaru</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="time_progressBar">
       <property name="value">
        <number>24</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="format">
        <string>%v z %m s</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="counts_tableWidget">
     <column>
      <property name="text">
       <string>Start</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Źródło</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Pkt.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Powt.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>A [CPS]</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>B [CPS]</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>C [CPS]</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>S [CPS]</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>AB [CPS]</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>BC [CPS]</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>AC [CPS]</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>T [CPS]</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>D [CPS]</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TDCR</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DT [%]</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rel.1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rel.2</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
